.github/workflows/tav.yml

```diff
@@ -0,0 +1,56 @@
+name: TAV
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths-ignore:
+    - '*.md'
+    - '*.asciidoc'
+    - 'docs/**'
+    - 'examples/**'
+  pull_request:
+    branches:
+    - main
+    paths-ignore:
+    - '*.md'
+    - '*.asciidoc'
+    - 'docs/**'
+    - 'examples/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+# limit the access of the generated GITHUB_TOKEN
+permissions:
+  contents: read
+
+jobs:
+
+  filter:
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    outputs:
+      matrix: ${{ steps.generate.outputs.matrix }}
+    steps:
+      - id: generate
+        uses: elastic/apm-pipeline-library/.github/actions/version-framework@current
+        with:
+          versionsFile: .ci/.jenkins_tav_nodejs.yml
+          frameworksFile: .ci/.jenkins_tav.yml
```

Comment:
How about putting the TAV module names and Node version name data in *this* file rather than having to have this separate action (that is using Python) to do this processing? Would that be possible/easier?  I think it would be nice to see the matrix values right here in the "tav.yml" file.

Reply:
No strong opinion, just trying to migrate without changing much the existing process. So, let me add them